src/app/demos/copy-to-clipboard/copy-to-clipboard-demo.component.html

```diff
@@ -0,0 +1,14 @@
+<div #divReference>
+  <h3>
```

Comment:
I think I would personally change the text here. The `Ctrl + C` doesn't really make sense to me as that is how you would normally copy without this button. I would probably change the `h3` to say something link "Text to copy" and then something like "All text within the wrapping `div` element will be copied to the clipboard!"

Reply:
✔️